Source/WebCore/platform/graphics/ByteArrayPixelBuffer.cpp

```diff
@@ -87,6 +87,13 @@ ByteArrayPixelBuffer::ByteArrayPixelBuffer(const PixelBufferFormat& format, cons
 {
 }
 
+Ref<JSC::Uint8ClampedArray> ByteArrayPixelBuffer::data(Ref<ByteArrayPixelBuffer>&& pixelBuffer)
```

Comment:
Sorry, realised this since we last spoke:
Here the `pixelBuffer` will still refer to the instance that is held by the original caller.
So we have:
```
Ref<ByteArrayPixelBuffer> myRef = getBuffer();
ByteArrayPixelBuffer::data(WTFMove(myRef));
ASSERT(myRef.isHashTableEmptyValue()); // This asserts, which is unexpected! We expected myRef to be moved from, but nothing did move construct from it..
```



So the signature must be:
```
Ref<JSC::Uint8ClampedArray> ByteArrayPixelBuffer::data(Ref<ByteArrayPixelBuffer> pixelBuffer) {
    if (pixelBuffer->hasOneRef()) // <-- this ref is the ref that the function holds and it lives until function ends.
```

Instead of this:
```
Ref<JSC::Uint8ClampedArray> ByteArrayPixelBuffer::data(Ref<ByteArrayPixelBuffer>&& pixelBuffer) {
    if (pixelBuffer->hasOneRef()) // <-- this ref is the ref that the caller holds and it lives as long as the caller wants.
```